gnocchi/incoming/__init__.py

```diff
@@ -124,12 +121,11 @@ def add_measures_batch(self, metrics_and_measures):
         :param metrics_and_measures: A dict where keys
         are metrics and value are measure.
         """
-        with futures.ThreadPoolExecutor(max_workers=_NUM_WORKERS) as executor:
-            list(executor.map(
-                lambda args: self._store_new_measures(*args),
-                ((metric, self._encode_measures(measures))
-                 for metric, measures
-                 in six.iteritems(metrics_and_measures))))
+        utils.parallel_map(
```

Comment:
so i'm thinking about memory usage here but does incoming need coordination? because this will load coordination code.

Reply:
It does not. I'll move coordination out.